components/script/dom/htmliframeelement.rs

```diff
@@ -578,6 +586,18 @@ impl VirtualMethods for HTMLIFrameElement {
                         modes
                     }));
             },
+            &local_name!("srcdoc") => {
+                // https://html.spec.whatwg.org/multipage/#the-iframe-element:the-iframe-element-9
+                // Whenever an iframe element with a non-null nested browsing context has its
+                // srcdoc attribute set, changed, or removed, the user agent must process the
+                // iframe attributes.
+
+                // < Q1 > Not sure whether following if statement is required
+                // if self.upcast::<Node>().is_connected_with_browsing_context() {
```

Comment:
Yes, this is necessary for the same reason that the code dealing with "src" modifications uses it.

Reply:
Done.